### DISCUSSION TITLE
Where is the storage location of the libraries?

### DISCUSSION THREAD

**Discussion by JonathanMeer** [jgraph/drawio-desktop#708] [Q&A]
Hi all
Does someone knwo where the storage location of the libraries is?
I would like to distribute my own libraries automatically throughout the company. 

Many thanks in advance
Johnny

**Comment by jepoyoy** [comment]
Hello. I'm also looking into this as well. I am trying to create a build of drawio for company internal, and would like to add custom library specifically for systems inside our company. Is that your use case as well @JonathanMeer ?
